source/extensions/all_extensions.bzl

```diff
@@ -15,6 +15,13 @@ def envoy_all_extensions(denylist = []):
     # These extensions can be removed on a site specific basis.
     return [v for k, v in all_extensions.items() if not k in denylist]
 
+_network_filter_prefix = "envoy.filters.network"
+
+def envoy_all_network_filters():
```

Comment:
Add a comment like the one above `envoy_all_extensions`, saying what this functions returns.

Reply:
Added! Thanks!